packages/storybook/.storybook/styles.scss

```
@@ -1,3 +1,2 @@
-@import '@faststore/ui/src/styles/base/utilities.scss';
-@import '@faststore/ui/src/styles/global.scss';
-@import '@faststore/ui/src/styles/components.scss';
+@import "@faststore/ui/src/styles/global.scss";
```

Stylelint (17.11.0)

[error] Unexpected extension ".scss" in `@import` (scss/load-partial-extension)

(scss/load-partial-extension)